api/proto/v1/teleworker.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+option go_package = "internal/api/v1";
+
+package v1;
+
+service TeleWorker {
+  rpc Start(StartRequest) returns (StartResponse);
+  rpc Stop(StopRequest) returns (StopResponse);
+  rpc Status(StatusRequest) returns (StatusResponse);
+  rpc Stream(StreamRequest) returns (stream StreamResponse);
+}
+
+// JobStatus represents a status of each job.
+// STARTING - the job was created, but has't started yet.
+// ALIVE - the job runs successfully at the moment.
+// FINISHED - the job finished its execution. 
+// STOPPED - the job was stopped by the user.
+enum JobStatus {
+  UNKNOWN = 0;
+  STARTING = 1;
+  ALIVE = 2;
+  FINISHED = 3;
+  STOPPED = 4;
+}
+
+// StartRequest is a request sent to start a job, contains:
+// a command line provided by user;
+// memory limit for the job in megabytes;
+// cpu weight percentage;
+// i/o weight percentage.
+message StartRequest {
+  string command_line = 1;
```

Comment:
Since you have a single string for the command, do you intend to run it in a shell or do something else? Please exec the given command directly, without a shell. In that case, a single string for the command and arguments may not be sufficient.

Reply:
Good catch, thanks! I did it the right way in the hardcoded prototype, but it completely got out of my head when I was thinking about the start request.
I've added optional args, updated the doc, and also added another usage example with piping commands to readme.